bears/python/requirements/PySafetyBear.py

```diff
@@ -74,10 +74,14 @@ def run(self, filename, file):
                 version_spec_match.end(1) + 1,
             )
 
+            try:
+                additional_info_temp = vulnerability.description
+            except KeyError:
+                additional_info_temp = ''
```

Comment:
you didnt implement https://github.com/coala/coala-bears/pull/2095#discussion_r147716949

Reply:
Only `vulnerability.description` is inside the `try` block i did't put entire `yield` inside the `try` block

Reply:
Or I have missed the `except` message that `description is renamed to advisory`

Reply:
If there is no `description` , you are using an empty string.
You are NOT (yet) using the new advisory field.